README.md

```diff
@@ -87,9 +87,17 @@ you must do.
         brew tap ArmMbed/homebrew-formulae
         brew install arm-none-eabi-gcc
 
+    On Windows Subsystem for Linux (WSL):
+
+        sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa
+        sudo apt-get update
+        sudo apt-get install gcc-arm-embedded
```

Comment:
`apt-get` is the old program, we should be using `apt`, unless WSL doesn't support it.

Reply:
This PR is not an update about `apt-get` to `apt`. I have kept apt-get to be consistent with Linux. I'll change both none-the-less.

Reply:
Oh,  I didn't realize the above stuff was wrong too. If you  fix it all, I'll give you a shiny emote:  ✨